modules/consul-cluster/main.tf

```diff
@@ -95,6 +95,7 @@ resource "aws_security_group" "lc_security_group" {
 }
 
 resource "aws_security_group_rule" "allow_ssh_inbound" {
+  count       = "${length(var.allowed_inbound_cidr_blocks) >= 1 ? 1 : 0}"
```

Comment:
Should this be `allowed_ssh_cidr_blocks`?

Reply:
Good catch. I've corrected this.